packages/safe-core-sdk/README.md

```diff
@@ -112,19 +105,23 @@ import Safe from '@gnosis.pm/safe-core-sdk'
 const safeSdk: Safe = await Safe.create({ ethAdapter: ethAdapterOwner1, safeAddress })
 ```
 
+Check the method `create` in the [API Reference](#sdk-api) for more details on additional configuration parameters.
```

Comment:
"method" should come after the name, i.e. "Check the `create` method..."

There are a few other instances of this in this file as well.